language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules_daml:daml.bzl", "daml_compile")
+load("@os_info//:os_info.bzl", "is_windows")
+
+daml_compile(
+    name = "daml2ts-test",
+    main_src = "daml/Main.daml",
+    srcs = glob(["daml/**/*.daml"]),
+)
+
+# This is needed to be able to call `$(location ...)` in rules.
+# Otherwise I have not found a way to express the conditional within the
+# `$(location ...)` substitution.
+alias(
+    name = "yarn",
+    actual = "@nodejs//:bin/yarn.cmd" if is_windows else "@nodejs//:bin/yarn",
+)
```

Comment:
This seems fine, and simpler than setting up a `yarn_dev_env` similar to `nodejs_dev_env`. Maybe `//3rdparty/nodejs:yarn` or `//:yarn` is a better place for this?

Reply:
I'll do this in a separate PR. There's an unrelated bazel file containing the same hack. I'll fix both then.